HISTORY.md

```diff
@@ -2,9 +2,16 @@
 
 ## UNRELEASED
 
+## v2.1.0 (2025-03-10)
+
+- Added support for ruff as default linting + formatting, updating gitignore to latest GitHub default (Discussion [#374](https://github.com/drivendataorg/cookiecutter-data-science/discussions/374) and Issue [#388](https://github.com/drivendataorg/cookiecutter-data-science/issues/388))
```

Comment:
```suggestion
- Changed linting and formatting to be a new configuration choice (Discussion [#374](https://github.com/drivendataorg/cookiecutter-data-science/discussions/374))
   - Added support for Ruff as a new option. 
   - Changed the default choice to be Ruff.
   - Changed the previous behavior to be named "flake8+black+isort".
- Fixed obsolete `[tool.ruff.lint.isort]` configuration key names that used underscores instead of hyphens. (Issue [#388](https://github.com/drivendataorg/cookiecutter-data-science/issues/388))
```

This should wait for #439.

Reply:
Just revised the suggestion with additional changes from #439.